compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -666,6 +666,12 @@ OMR::Z::CodeGenerator::CodeGenerator()
    self()->setSupportsBCDToDFPReduction();
    self()->setSupportsIntDFPConversions();
 
+   // On 31-Bit zOS/zLinux We rely on optimizer to generate array copy trees specialized for direction
```

Comment:
@fjeremic  I have opened up issue on OMR https://github.com/eclipse/omr/issues/3229 to track and fix issue for 31-Bit platform.